docs/getting-started/getting-started-with-globus.md

```diff
@@ -0,0 +1,38 @@
+---
+title: User Guide - Transferring Data to RCIF_CHPC using Globus
+created: 2024-10-25
+tags: []
+permalink: /getting-started/globus/
+author: dptru10
+include: true
+---
+
+## Globus Overview
+
+Globus is a secure, reliable data transfer service designed for research and scientific computing. It allows users to move large datasets between institutions, labs, and computing facilities.
+
+## Using Globus Personal Connect
+
+1. Download and install Globus Personal Connect on your local machine from [Globus Connect Personal](https://www.globus.org/globus-connect-personal)
+2. Set up an endpoint on your computer to enable file transfers.
+
+## Transferring Data to RCIF_CHPC
+
+1. Log in to the [Globus web interface](https://app.globus.org)
```

Comment:
Could you also add some more detail? For instance, it may not be obvious to a user that they should use the WUSTL SSO to authenticate or that permissions must be granted.